source/PlayerInfoPanel.cpp

```diff
@@ -545,7 +555,34 @@ void PlayerInfoPanel::DrawPlayer(const Rectangle &bounds)
 	DrawList(licenses, table, "licenses:", maxRows, false);
 }
 
+bool CompareName(const shared_ptr<Ship> &lhs, const shared_ptr<Ship> &rhs) {
```

Comment:
> The player is not allowed to reorder their ships while in flight. With this PR, because it operates on the player's ship list directly, this requirement is no longer met.

I can see a few reasons for this but I'd like to hear from your side why this is so.

> Personally, I would prefer to have some kind of "save" or "record" button that establishes the current order as the recorded ship order, but if not clicked, the order reverts to the previous one. There's lot of times that I want to sort by this or that just to count ships or check on something, and in those instances I do _not_ want it to save the order.

I think this is the most non-invasive way to do it. However, if the player chooses to reorder their ships themselves while docked, then should I assume the list should be saved?

Or should it be the other way? Having a reset button when the sort doesn't go the way you want. I agree the sorting action shouldn't have immediate permanent consequences, or at least there should be an easy way to undo it.